clip.py

```diff
@@ -173,7 +197,7 @@ def writeData(self, clip, channel, offset, data):
             #                % (offset, data.shape[0], self.length(clip)))
 
         self.data[clip.audio_file][offset:offset + data.shape[0],
-                                   channel] = data
+        channel] = data
```

Comment:
@Vampouille may I introduce a new temporary variable `end = offset + data.shape[0]`directly after the `clip.audio_file is None` check?
That would circumvent the line break and possibly make the code more readable (glad to hear a name more expressive than *end*, couldn't think of anything better quickly)